Leibit.Core/Data/sued/TF.xml

```diff
@@ -0,0 +1,412 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<estw>
+  <station name="Radolfzell" short="RRZ" refNr="56">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="66BP" direction="right" />
+      <block name="56BN2" direction="right" />
+    </track>
+  </station>
+
+  <station name="Schwackenreute" short="RSU" refNr="65">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="65BN" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Eriskirch" short="TEK" refNr="76">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="77BP2" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Baienfurt" short="TBAG" refNr="40">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="40BN2" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Aulendorf" short="TAU" refNr="38">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="38BP69" direction="left"/>
+      <block name="39BP" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Brandbühl" short="RBRB" refNr="66" scheduleFile="g66_____.abf" localOrderFile="bf66____.abf">
```

Comment:
Die Betriebsstelle heißt "Üst Brandbühl".

Reply:
Ich hab nochmal meine beiden FDL Kollegen gefragt. Beide meinten im Normalfall heißen die Betriebsstellen ohne Hp oder Üst. Die Zusätze werden nur genutzt wenn es eine doppelte Verwendung gibt. Beispiel dafür in Lindau: Lindau-Aeschach; dort gibt es einen Hp und den Abzw. Der Hp wird dann als "Lindau-Aeschach (Hp)" und der Abzweig bleibt ohne Zusatz.
Soll ich es anpassen damit alles einheitlich ist, oder wollen wir das bei den anderen Stellwerken hin zur DB anpassen?

Reply:
Da habe ich wieder was gelernt :) Dann lass es ruhig so.